docker/cmd/run.sh

```diff
@@ -2,11 +2,13 @@
 
 sleep 1
 
+chmod +x docker/cmd/run.sh
```

Comment:
Sorry, I think I didn't express it well. I just meant this file `run.sh` itself, IMO should be `chmod +x`, to grant the execution permission to users.